src/globals/scss/core/_typography.scss

```diff
@@ -157,4 +157,10 @@
     }
   }
 
+  // Form group error style
+  .govuk-form-group--error {
```

Comment:
This is not related to typography in any way. I am not sure where it should live (sorry!) but it's definitely not here.